pkg/kb-summarizer/report/report.go

```diff
@@ -28,12 +30,19 @@ const (
 )
 
 type Check struct {
-	ID          string     `yaml:"id" json:"id"`
-	Text        string     `json:"description"`
-	Remediation string     `json:"remediation"`
-	State       State      `json:"state"`
-	NodeType    []NodeType `json:"node_type"`
-	Nodes       []string   `json:"nodes,omitempty"`
+	ID             string      `yaml:"id" json:"id"`
+	Text           string      `json:"description"`
+	Remediation    string      `json:"remediation"`
+	State          State       `json:"state"`
+	NodeType       []NodeType  `json:"node_type"`
+	Nodes          []string    `json:"nodes,omitempty"`
+	Audit          string      `json:"audit"`
+	AuditConfig    string      `yaml:"audit_config"`
+	TestInfo       []string    `json:"test_info"`
+	Commands       []*exec.Cmd `json:"commands"`
+	ConfigCommands []*exec.Cmd `json:"config_commands"`
+	ActualValue    string      `json:"actual_value"`
+	ExpectedResult string      `json:"expected_result"`
```

Comment:
Is there a reason why the Go tags are `yaml`, `json`, or both for different fields?

If there is a distinction, I think we should move `AuditConfig` to the top since it's the only YAML field.

Reply:
I changed them to all json, they should not be yaml.